include/ibm/management_console_rest.hpp

```diff
@@ -69,18 +102,63 @@ inline void saveConfigFile(const std::string& data, const std::string& fileID,
         asyncResp->res.result(boost::beast::http::status::bad_request);
         asyncResp->res.jsonValue["Description"] =
             "File size is less than minimum allowed size[100B]";
-        return;
+        return false;
     }
     if (data.length() > maxSaveareaFileSize)
     {
         asyncResp->res.result(boost::beast::http::status::bad_request);
         asyncResp->res.jsonValue["Description"] =
             "File size exceeds maximum allowed size[10MB]";
-        return;
+        return false;
     }
     std::ofstream file;
     std::filesystem::path loc(configFilePath);
 
+    // Get the current size of the savearea directory
+    std::filesystem::recursive_directory_iterator iter(loc, ec);
+    if (ec)
+    {
+        asyncResp->res.result(
+            boost::beast::http::status::internal_server_error);
+        asyncResp->res.jsonValue["Description"] = internalFileSystemError;
+        BMCWEB_LOG_DEBUG << "handleIbmPut: Failed to prepare save-area "
```

Comment:
Why not ERROR?

Reply:
I will take it in a new pr and change the debug logs to error logs wherever it is necessary.. This includes not only save area but also locks..